docs/src/reference/cli/changelog.rst

```diff
@@ -1,6 +1,11 @@
 CLI Changelog
 =============
 
+1.0.3 (2020-12-16)
+------------------
+
+* Fixes "Invalid constructor input" errors in verify, delete, and message commands.
```

Comment:
I blame myself for this one because i didnt do a good job following the pattern below, but if you could format this to follow he format with the fixed heading that would be great!:

Changelog
=========

1.0.3 (2020-12-16)
------------------

Fixed
*****

* Fixes "Invalid constructor input" errors in verify, delete, and message commands.

Reply:
I'm a little confused - should I remove the bullet points?